packages/strapi-plugin-graphql/services/Query.js

```diff
@@ -254,13 +258,19 @@ module.exports = {
         return policy;
       }
 
+      // Initiliase loaders for this request.
+      Loaders.initializeLoader();
+
       // Resolver can be a function. Be also a native resolver or a controller's action.
       if (_.isFunction(resolver)) {
         context.query = this.convertToParams(options);
         context.params = this.amountLimiting(options);
 
+        // Avoid population.
+        context.query._populate = model.associations.filter(a => !a.dominant && _.isEmpty(a.model)).map(a => a.alias);
+
         if (isController) {
-          const values = await resolver.call(null, context);
+          const values = await resolver.call(null, ctx);
```

Comment:
Wait what :O How are you retrieving the params and queries passed to the service?